src/Symfony/Bundle/FrameworkBundle/Resources/config/error_renderer.php

```diff
@@ -30,6 +30,7 @@
                     ->factory([HtmlErrorRenderer::class, 'getAndCleanOutputBuffer'])
                     ->args([service('request_stack')]),
                 service('logger')->nullOnInvalid(),
```

Comment:
I don't understand the sentence. `an` + exception`s`. Did you mean`a list of exceptions`?
same for constructor docblock

Reply:
I took it from [here](https://github.com/symfony/symfony/blob/6.1/src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php#L108) and I also did not understand it first time when I read it, but now I think I understand it like "an (exceptions to log & status code) mapping", so the article refers to "mapping". Sorry, English is not my mother tongue, so I took it from the same parameter passed to exception listener, but happy to change the wording. Question: if we decide to change, should I do that in the exception listener DI config as well?

Reply:
What do yo think of "Configuration per exception class"?

Reply:
I'm fine with that description, but let me know if I should change the original usages of it as well

Reply:
I mean for me it's hard to say now as I understand what it's all about and any of these descriptions is clear. However, for someone who sees it for the first time probably the latter one is better. But again, for me it's hard to say now (and I'm not a native English speaker)

Reply:
Same feeling for me.
You can update on both places. Thanks.

Reply:
done, thanks!